o-fish-ios/Views/Components/PatrolBoatView/BottomPatrolView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  BottomPatrolView.swift
+//  
+//  Created on 9/8/20.
+//  Copyright © 2020 WildAid. All rights reserved.
+//
+
+import SwiftUI
+
+struct BottomPatrolView: View {
+    @Binding var draftBoardingsCount: Int
+    var findAction: (() -> Void)?
+    var boardVesselAction: (() -> Void)?
+    var draftBoardingsAction: (() -> Void)?
+
+    private enum Dimensions {
+        static let height: CGFloat = 83.0
+        static let heightEllipse: CGFloat = 42.0
+        static let bottomPadding: CGFloat = 78.0
+    }
+
+    var body: some View {
+        HStack {
+            Spacer()
+
+            HStack(alignment: .bottom) {
+
+                FindRecordsButtonView(action: findAction)
+                    .padding(.horizontal)
+                Spacer()
+
+                BoardVesselButtonView(action: boardVesselAction)
+                    .padding(.horizontal)
+                Spacer()
+
+                DraftBoardingsButtonView(draftBoardingsCount: $draftBoardingsCount,
+                                         action: draftBoardingsAction)
+                    .padding(.horizontal)
+
+            }
+                .padding(.bottom, Dimensions.bottomPadding)
+            Spacer()
+        }
+        .frame(height: Dimensions.height)
+        .background(Color.white)
+    }
+}
+
+struct BottomPatrolView_Previews: PreviewProvider {
+    static var previews: some View {
+        BottomPatrolView(draftBoardingsCount: .constant(5))
+    }
+}
```

Comment:
Can use the previews to see how the view looks in different locales:

```
struct BottomPatrolView_Previews: PreviewProvider {
    static let localizations = Bundle.main.localizations.map(Locale.init).filter { $0.identifier != "base" }

    static var previews: some View {
        Group {
            ForEach(localizations, id: \.identifier) { locale in
                BottomPatrolView(draftBoardingsCount: .constant(5))
                    .environment(\.locale, locale)
                    .previewDisplayName(Locale.current.localizedString(forIdentifier: locale.identifier))
            }
        }
    }
}
```

Reply:
Fixed